ssm-client.spec

```diff
@@ -118,7 +118,7 @@ if [ $1 -gt 1 ] || [ -f /usr/local/percona/pmm-client/pmm.yml ]; then
         cp /usr/local/percona/pmm-client/server.key /opt/ss/ssm-client/server.key
     fi
     if [ -d /usr/local/percona/qan-agent ]; then
-        find /usr/local/percona/qan-agent -maxdepth 1 ! -name bin -exec cp -r "{}" /opt/ss/ssm-client/qan-agent/ \;
+        find /usr/local/percona/qan-agent -maxdepth 1 ! -path /opt/ss/percona/qan-agent ! -name bin -exec cp -r "{}" /opt/ss/qan-agent/ \;
```

Comment:
/percona/ ?

Reply:
Fixed, I must have copied that line from deb packaging files.